docs/src/modules/components/MarkdownElement.js

```diff
@@ -170,7 +172,8 @@ const styles = (theme) => ({
     },
     '& a, & a code': {
       // Style taken from the Link component
-      color: theme.palette.primary.main,
+      color:
+        theme.palette.mode === 'dark' ? theme.palette.primary.main : theme.palette.primary.dark,
```

Comment:
This looks wrong, compare the two links:

<img width="456" alt="Capture d’écran 2021-06-05 à 20 33 22" src="https://user-images.githubusercontent.com/3165635/120901822-4ce03980-c63d-11eb-816c-99c25af2a2ce.png">

Related to #26145

I would expect

```suggestion
      color: theme.palette.primary.main,
```

Reply:
My mistake, I want to fix only the `a code` because contrast does not good enough. it's fixed.

![image](https://user-images.githubusercontent.com/18292247/120926872-e1fc3480-c708-11eb-8461-3e6dd2c998e6.png)